app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
It's actually not method on host, it's a helper on the ActionView.  When using hostgroup provisioning without this allowed in the Jail you get this error:

```
There was an error rendering the Unnamed template: The snippet
'subscription_manager_registration' threw an error: Safemode
doesn't allow to access 'subscription_manager_configuration_url' on
#<Safemode::ScopeObject>
```

With this fix the template renders fine.